server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/actor/Actor.java

```diff
@@ -0,0 +1,92 @@
+package de.tum.in.www1.hephaestus.codereview.actor;
+
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.hephaestus.codereview.comment.Comment;
+import de.tum.in.www1.hephaestus.codereview.pullrequest.Pullrequest;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.OneToMany;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "actors")
+@Getter
+@Setter
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class Actor {
```

Comment:
For now I wanted to stick to the most important attributes, I personally would just add everthing "on-demand" to not spam our database. Would you agree with that approach?

Totally forgot about about the `name` as attribute though, just added it in 05ed344.

We don't really need the `avatar_url` explicitly as it as available on Github via `{profile_url}.png`

Reply:
I would have just stored the `avatar_url` it might be different for GitLab and now the client needs to know that the avatar is located at `{profile_url}.png`. We don't have to be that frugal with the data we are storing.